src/Form.js

```diff
@@ -0,0 +1,689 @@
+import React from "react";
+// import FormSection from "./FormSection";
+
+class Form extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			userTitle: "",
+			userFirstName: "",
+			userLastName: "",
+			userAddressLine1: "",
+			userAddressLine2: "",
+			userAddressCountry: "",
+			userAddressCityMunicipality: "",
+			userAddressProvinceTerritory: "",
+			userAddressCode: "",
+			userContactNumber: "",
+			userContactEmail: "",
+			userContactConfirmEmail: "",
+			userContactLanguage: "",
+			userSpammedNumber:"",
+			userSpammedPersonal:false,
+			userSpammedBusiness:false,
+			userSpammedProvider:"",
+			callUserAction:"",
+			callOnPickup:"",
+			callNameOrNumber:"",
+			callDeclaration:"",
+			callerNumber: "",
+			callerID: "",
+			callerAddress: "",
+			callerWebsite: "",
+			callDate: "",
+			callTime: "",
+			reasonSell:false,
+			reasonWon:false,
+			reasonSurvey:false,
+			reasonPolitical:false,
+			reasonDonation:false,
+			reasonDebt:false,
+			reasonPsa:false,
+			reasonUnknown:false,
+			purchaseStatus:"",
+			purchaseNow:false,
+			purchaseRecent:false,
+			purchaseDistant:false,
+			businessInquiry:false,
+			businessContract:false,
+			businessDivulge:false,
+			businessVisit:false,
+			ceaseNow:false,
+			ceasePrior:false,
+			ceaseRefuse:false,
+			consent:false
+		};
+	}
+
+	handleChange = (event) => {
+		const { name, value , checked, type} = event.target;
+		type==="checkbox" ?
```

Comment:
Whoa I've never seen a ternary operator used in this way interesting. I've always seen it used for computing a value, not determining which action to do.
Kinda neat!
But beware, this isn't common so ppl might look at ya funny XD

Reply:
tbh, it might be clearer to have 2 separate methods, handleChangeInput, hasChangeCheckBox